### PR TITLE
Update to new edalize

### PR DESCRIPTION
**Pull request by mtdudek**
Update yosys-uhdm to newer version

Signed-off-by: Maciej Dudek <mdudek@antmicro.com>

### CODE REVIEW

File: conf/common/requirements.txt [hunk 1]
```diff
@@ -10,4 +10,4 @@ termcolor
 terminaltables
 tqdm
 yapf
-git+https://github.com/SymbiFlow/edalize.git@symbiflow#egg=edalize
+git+https://github.com/antmicro/edalize.git@symbiflow_update#egg=edalize
```
Comment (mithro) [inline conf/common/requirements.txt:13]: Can you just push to the SymbiFlow repo?

**Review by mithro — COMMENTED** [review]
It looks like you are making some change around `EDALIZE_LAUNCHER` in some way?

File: conf/common/requirements.txt (hunk 1 above)
Comment (acomodi) [reply conf/common/requirements.txt:13]: This PR is mainly to verify that the rebased version of edalize is working. As soon as it is verified to work we'll push to symbiflow and get the upstream PRs landed: https://github.com/olofk/edalize/pull/262, https://github.com/olofk/edalize/pull/261, https://github.com/olofk/edalize/pull/260

**Review by acomodi — APPROVED** [review]
LGTM as soon CI goes green

**Pull request merged**